### PR TITLE
ISPN-10018 Coarse-grained logging categories

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-10018

### CODE REVIEW

**Comment by tristantarrant** [comment]
Before I rebase, I would like to know if everyone agrees with the categories. This PR is a cat and mouse game and I'd like to rebase the least number of times possible

**Comment by tristantarrant** [comment]
Rebased

**Comment by ryanemerson** [comment]
> Before I rebase, I would like to know if everyone agrees with the categories. This PR is a cat and mouse game and I'd like to rebase the least number of times possible

What's the plan for counters and multimap? Do they get their own category or just CONTAINER?

Also, is the plan to add PERSISTENCE in this PR or a follow up?

**Comment by tristantarrant** [comment]
Yes, CONTAINER for counters, locks and multimaps.
PERSISTENCE is in  org.infinispan.util.logging.Log. Categories need to be redefined in the various loggers because we have dropped logger inheritance to reduce class size.

**Comment by tristantarrant** [comment]
Added counter and query modules

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

File: core/src/main/java/org/infinispan/configuration/parsing/XmlConfigHelper.java [hunk 1]
```diff
@@ -118,9 +116,6 @@ public static Object valueConverter(Class klass, String value) {
             continue;
          }
          catch (NoSuchMethodException me) {
-            // this is ok, but certainly log this as a warning
-            // this is hugely noisy!
-            //if (log.isWarnEnabled()) log.warn("Unrecognised attribute " + propName + ".  Please check your configuration.  Ignoring!!");
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/configuration/parsing/XmlConfigHelper.java]: I think you can delete the whole method

**Pull request merged**